test-result-summary-client/src/Build/AllTestsInfo.jsx

```diff
@@ -84,7 +84,10 @@ const Build = () => {
             return rt;
         });
 
-        setParents(fetchedParents);
+        // Filter out siblings builds (Build #77 - 74)
+        const filteredParents = fetchedParents.filter(parent => parent.buildNum > 77); 
```

Comment:
We need to have a generic solution. Not for any specific build numbers. 
In this case, I think we do not need parents data.
To quickly test this, we can try to remove `parents={parents}` at https://github.com/adoptium/aqa-test-tools/blob/bff112de9d93ef5f6674c7235875e582517e1c35/test-result-summary-client/src/Build/AllTestsInfo.jsx#L167 and confirm the behavior. Then we can remove parents data completely.

Reply:
i have removed `parents={parents}` part from line 167.

Reply:
removing `parents={parents}` was suggested to be a quick test, not the full solution.
Currently, removing `parents={parents}` results the allTestsInfo page to crash `TypeError: Cannot read properties of undefined (reading 'buildName')`. We need to investigate more.

Reply:
oh okay i understood it now, i would put the `parents={parents}` back and try to fix the other parts. But i have a question of is there are another page like https://trss.adoptium.net/allTestsInfo?buildId=66582ddf879917006e217d83&limit=5&hasChildren=true where I could also test on? Thank you!

Reply:
@Sangyoon21 Yes, it would be the case like the page you posted. We just want to show the first build (`78` `Failed` in this case, but not 77  7 6 75 etc.).

Reply:
For the test, in your local TRSS, you can add monitoring of this build `evaluation-openjdk17-pipeline`(you can refer the current monitoring list for the format), the data will shown after downloaded to your local TRSS.

Reply:
Okay thank you for the response! i would try to fix this!